src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -33,6 +35,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Utilities" Version="$(MicrosoftVisualStudioUtilitiesVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" Version="$(MicrosoftVisualStudioValidationVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.CoreUtility" Version="15.8.39-preview-g35168dac43" />
```

Comment:
I thought the plan was to target 15.7

Reply:
That's the version @AmadeusW told me to use :).

Reply:
we'll update it once we have a public nuget with `ImageId`

Reply:
It should be already on myget in 15.6.253 bits (which contain both Quickinfo and commanding changes). https://vside.myget.org/feed/vssdk/package/nuget/Microsoft.VisualStudio.CoreUtility/15.6.253-preview